src/chunk_append/exec.c

```diff
@@ -333,18 +333,41 @@ chunk_append_exec(CustomScanState *node)
 		 * more subplans, return the empty slot set up for us by
```

Comment:
the part of the comment about returning empty slot is now out of date? (happens above?)